multilang: Use custom font instead of Unifont

eFiniLan opened the pull request:
- Replace unifont.otf with [OpFont](https://github.com/eFiniLan/opfonts) (IBM Plex based, 4 weights) rendered at 48px.
- Generate per-language atlases (OpFont-{weight}-{lang}) instead of one giant atlas.
- Generate a labels atlas (OpFont-{weight}-Labels) for the language selector.
- on_language_changed(): preloads new fonts, remaps stale widget references, then unloads old fonts.
- Fonts load lazily per weight instead of all at startup.

eFiniLan commented:
actually, the unifont looks OKish on the actual device, I will close now :)

adeebshihadeh commented:
Hey @eFiniLan, thanks for the PRs.

We're busy getting the fours out the door. We'll get to this soon after that.

jyoung8607 commented:
Marked ignore-stale to make sure this sticks around until Chinese localization quality can be looked at.

eFiniLan committed:
multilang: use custom-built font instead of unifont

Replace unifont with per-language OpFont atlases for better CJK/multilingual
rendering. Fonts are processed at multiple weights with language-specific
glyph subsets. Point LFS to fork GitHub storage.

adeebshihadeh commented:
Sorry this took so long @eFiniLan, but other languages should have good fonts now!

I chose Noto since it's closest to Inter.
